src/RotativaCore.SampleWebApp.3.0.Linux/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM mcr.microsoft.com/dotnet/core/aspnet:3.0-buster-slim AS base
+WORKDIR /app
+EXPOSE 80
+EXPOSE 443
+
+### Customize for your environment. ###
+RUN apt-get update -y && apt-get install -y \
+  wget \
+  libssl-dev \
+  libxrender1 \
+  libxext6 \
+  libfontconfig1 \
+  fontconfig \
+  xfonts-75dpi \
+  xfonts-base \
+  libjpeg62-turbo
```

Comment:
Don't forget `libssl1.0-dev` if you want CSS.

Reply:
Thank you. On Moby Linux VM (Docker for Windows) or the latest Debian, it specifies libssl-dev not libssl1.0-dev. 
It's the job of the administrator to make wlhtmltopdf work correctly in the use environment. Oh, So LINUX.

Reply:
hehe true :)